CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing
+Oracle welcomes contributions to this repository from anyone.
+
+If you want to submit a pull request to fix an bug or enhance an existing
+`Dockerfile`, please first open an issue and link to that issue when you
+submit your pull request.
+
+If you have any questions about a possible submission, feel free to open
+an issue too.
+
+## Pull Request Process
+1. Fork this repository
+2. Create a branch in your fork to implement the changes. We recommend using
+the issue number as part of your branch name, e.g. `1234-fixes`
+3. Ensure that any documentation is updated with the changes that are required
```

Comment:
There should be another point saying something like "Ensure that samples are updated if the PR causes the behavior of the sample to change"